submissions/RubyAdeniregun/HTML/Product Page.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>DUNA STORES</title>
+</head>
+<body>
+   <hi><b>DUNA STORES</b></hi> 
+   <nav>
+       <ul>
+           <li><a href="#">Home</a></li>
+           <li><a href="#">Back</a></li>
+           <li><a href="#">Categories</a></li>
+       </ul>
+   </nav>
+   <hr>
+   <section>
+    <h2>Fresh Body Wash (Organic and Skin Brightening Black Soap)</h2>
+    <h3><em>Brand: Skin Perfection Organics</em></h3>
+   </section>
+
+   <!--This section is for product details-->
+   <section>
+       <h3>Product Details</h3>
+       <img src="C:\Users\CRM ABUJA\Desktop\DUFUNA\Images\IMG-20201001-WA0129.jpg" alt="Fresh Body Wash" style="width: 200px; height: 150px;">
```

Comment:
I can see that your Images folder is completely outside your project. Kindly create the Images folder within your project. You can then edit the img src appropriately.

Reply:
Done. Relocated images folder into submissions folder in the Dufuna Fem Project Folder and changed img file path